lib/base_wizard.py

```diff
@@ -186,12 +190,25 @@ def choose_hw_device(self):
                 continue
             devices += list(map(lambda x: (name, x), u))
         if not devices:
-            msg = ''.join([
+            msgs = [
                 _('No hardware device detected.') + '\n',
-                _('To trigger a rescan, press \'Next\'.') + '\n\n',
-                _('If your device is not detected on Windows, go to "Settings", "Devices", "Connected devices", and do "Remove device". Then, plug your device again.') + ' ',
-                _('On Linux, you might have to add a new permission to your udev rules.'),
-            ])
+                _('To trigger a rescan, press \'Next\'.') + '\n\n'
+            ]
+
+            if sys.platform in ('win32', 'win64', 'windows'):
```

Comment:
Yes.  Nice.  Why am I hearing about Windows when I'm on Linux or macOS? Etc? Python knows this stuff already.  That always kind of bugged me.  Thanks for this.